_saas-integrations/netsuite/v1/netsuite-v1.md

```diff
@@ -0,0 +1,426 @@
+  - title: "add integration"
+    content: |
+      4. In the **Account** field, enter the {{ integration.display_name }} account ID you retrieved in [Step 7](#locate-netsuite-account-id).
+      5. In the **Consumer Key** field, paste the Consumer Key you generated when you [created Stitch's integration record](#create-stitch-integration-record).
+      6. In the **Token ID** field, paste the Token ID you generated when you [created Stitch's access tokens](#create-access-tokens).
+      7. In the **Consumer Secret** field, paste the Consumer Secret you generated when you [created Stitch's integration record](#create-stitch-integration-record).
+      8. In the **Token Secret** field, paste the Token Secret you generated when you [created Stitch's access tokens](#create-access-tokens).
+  - title: "historical sync"
+  - title: "replication frequency"
+  - title: "track data"
+
+
+# -------------------------- #
+#     Replication Details    #
+# -------------------------- #
+
+replication-sections:
+  - content: |
+      In this section:
+
+      {% for section in page.replication-sections %}
+      - [{{ section.title | flatify }}](#{{ section.anchor }})
+      {% endfor %}
+
+  - title: "Custom records"
+    anchor: "custom-records"
+    content: |
+      For each custom record type in {{ integration.display_name }}, a table for that custom record type will be available for selection in Stitch.
+
+    subsections:
+      - title: "Table names for custom record types"
+        anchor: "custom-record-types-table-names"
+        content: |
+          Custom record tables are named `customrecord_[custom_record_type]`, where `[custom_record_type]` is the name of the custom record in {{ integration.display_name }}. For example: If a custom record were named `promo discount` in {{ integration.display_name }}, the corresponding table for those records would be named `customrecord_promo_discount`.
+
+      - title: "Replication methods for custom record types"
+        anchor: "custom-record-type-replication"
+        content: |
+          {% include layout/image.html type="right" file="/integrations/netsuite-custom-record-bookmarks.png" alt="Highlighted Show Creation Date and Show Last Modified fields in NetSuite's Custom Record Type page" max-width="350" %}
+          The Replication Method Stitch uses to replicate data for a custom record type depends on two custom record settings in {{ integration.display_name }}: **Show Last Modified** and **Show Creation Date**. These settings determine whether the {{ integration.display_name }} SuiteTalk API will return timestamp columns to use as [Replication Keys]({{ link.replication.rep-keys | prepend: site.baseurl }}).
+
+          - **If the record definition has either setting enabled**, Stitch will use [Key-based Incremental Replication]({{ link.replication.key-based-incremental | prepend: site.baseurl }}). This means that only new and updated records for the record type will be replicated during each job.
```

Comment:
We'll only use Incremental if "Last Modified" is set. We don't have any special logic to use the creation date, since we can't determine this setting from the API. NetSuite just returns Null for those fields if they aren't checked.

Checking CreationDate will populate that column in the destination.